app/src/main/java/org/mozilla/fenix/home/blocklist/BlocklistHandler.kt

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.blocklist
+
+import androidx.annotation.VisibleForTesting
+import mozilla.components.support.ktx.kotlin.sha1
+import org.mozilla.fenix.home.recentbookmarks.RecentBookmark
+import org.mozilla.fenix.home.recenttabs.RecentTab
+import org.mozilla.fenix.home.recentvisits.RecentlyVisitedItem
+import org.mozilla.fenix.utils.Settings
+
```

Comment:
Will do! Wanted to push the implementation before EoD yesterday in case you were able to take a look this morning, but will put all the finishing touches like that together today